README.md

```diff
@@ -234,6 +235,21 @@ const customEmojis = [
 <Picker custom={customEmojis} />
 ```
 
+## Not Found
+You can provide a custom Not Found object which will allow the appearance of the not found search results to change. In this case, we change the default 'sleuth_or_spy' emoji to Octocat when our search finds no results.
+
+```js
+import { Picker } from 'emoji-mart'
+
+const notFound = {
+  {
+    notFoundImg: () => <img src='https://assets-cdn.github.com/images/icons/emoji/octocat.png?v7' />
```

Comment:
Should we remove the `Img` part of this key now? I wonder if we should just make `notFound` take the function. 

So usage would be: 

```js
<Picker notFound={() => <img src='https://assets-cdn.github.com/images/icons/emoji/octocat.png?v7' />} />
```